src/mon/FSCommands.cc

```diff
@@ -295,16 +295,17 @@ class SetHandler : public FileSystemCommandHandler
         });
       }
     } else if (var == "balancer") {
-      if (val.empty())
+      if (val.empty()) {
         ss << "unsetting the metadata load balancer";
-      else 
+      } else {
         ss << "setting the metadata load balancer to " << val;
         fsmap.modify_filesystem(
             fs->fscid,
             [val](std::shared_ptr<Filesystem> fs)
         {
           fs->mds_map.set_balancer(val);
         });
+      }
```

Comment:
Oh, sorry, I think we don't need the braces here. The problem is from line 302 to line 307. We can reduce 2 indents from line 302 to line 307 (fsmap.modify_filesystem). :-)